.travis.yml

```diff
@@ -21,10 +21,10 @@ deploy:
     tags: true
     python: 3.6
     branch: master
-- provider: script
-  script: bash scripts/deploy.sh
-  on:
-    branch: master
+# - provider: script
+#   script: bash scripts/deploy.sh
```

Comment:
doesnt this mean that we should remove the deploy file as well? or do you want to keep that?

Reply:
@carocad you're right. I'll remove it. Wanted to keep it as a recipe but it is unnecessary clutter.